pom.xml

```diff
@@ -37,6 +37,12 @@
             <version>${webdrivermanager.version}</version>
             <scope>test</scope>
```

Comment:
- [ ] only thing required is remove this line. 
then the next dependency is not needed.